### PR TITLE
chore(web): use react-scripts for web bundle

### PR DESCRIPTION
**Pull request by r17x**
Currently we use `basic webpack configuration` by `bsb init` with `react-hooks template`. I think, we can use react-scripts for `small bundle` in current `web application`.

### Step to `Migrate`

- [x] remove webpack (with related dependencies as plugins)  https://github.com/evilfactorylabs/gow/pull/13/commits/ef117e5bdc5104d5c351802b39953c7621712bba
- [x] add `react-scripts` https://github.com/evilfactorylabs/gow/pull/13/commits/7eed88de024c26c8cb2356a07efb35a09fc246c8
- [x] make `project structure` compatible with `react-scripts` https://github.com/evilfactorylabs/gow/pull/13/commits/8c23120f5b5dc432951dcb77504e686d261a1607

### Maintain
- [x] upgrade dependencies https://github.com/evilfactorylabs/gow/pull/13/commits/8c23120f5b5dc432951dcb77504e686d261a1607
- [x] etc (chore things) https://github.com/evilfactorylabs/gow/pull/13/commits/7eed88de024c26c8cb2356a07efb35a09fc246c8
- [x] refactor: need changes after `upgrade dependencies` https://github.com/evilfactorylabs/gow/pull/13/commits/1c4bb1bfd82cb440ea911d88b10097c997f7acef
- [x] improve  `docker build` (reduce time build in golang project) https://github.com/evilfactorylabs/gow/pull/13/commits/b2a0bd5a573dfc849ce3a76bb901c18be6425c7d
 
### Before
image
### After 
image

```bash

File sizes after gzip:

  120.09 KB  build/static/js/2.aed0ec68.chunk.js
  3.17 KB    build/static/js/main.4d614ac1.chunk.js
  783 B      build/static/js/runtime-main.4ac9ccea.js
```

### Docker Build
image

### CODE REVIEW

**Review by faultables — APPROVED** [review]
LGTA

File: web/public/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>%REACT_APP_NAME%</title>
+  </head>
+  <body>
+    <div id="g0W">
```
Comment (faultables) [inline web/public/index.html:27]: Konspirasi

Comment (r17x) [reply web/public/index.html:27]: Konspirasi gmn sih 😏. Check docker yg copy .env

Comment (faultables) [reply web/public/index.html:27]: Namanya gak `root` kayak di tutorial2

Comment (r17x) [reply web/public/index.html:27]: Tutorial it's an example why not to use our naming

Comment (faultables) [reply web/public/index.html:27]: nvm bro

Comment (r17x) [reply web/public/index.html:27]: Artinya?

**Pull request merged**